src/test/java/core/basesyntax/fileservice/impl/CsvFileReaderServiceImplTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.fileservice.impl;
+
+import core.basesyntax.fileservice.CsvFileReaderService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvFileReaderServiceImplTest {
+    private final CsvFileReaderService csvFileReaderService = new CsvFileReaderServiceImpl();
```

Comment:
```suggestion
    private final CsvFileReaderService csvFileReaderService;
```
initialize in before class method